sheets/pod

```diff
@@ -5,7 +5,8 @@
 # Create a Podfile for the current project with the default contents:
 pod init
 
-# Download and install all pods defined in the Podfile (that haven't been installed before):
+# Download and install all pods defined in the Podfile
+# (that haven't been installed before):
```

Comment:
It's okay to split up parenthesized text. Also, remember you can substitute _and_ for the ampersand ('&'), which can be useful in a pinch.